### PR TITLE
search

### PR DESCRIPTION
**Pull request by teh**

### CODE REVIEW

File: basics/searching-packages.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Searching packages
+
+The simplest way to find a package is to use the web frontend on the
+NixOS website: https://nixos.org/nixos/packages.html
+
+On the command line the `nix-env` command supports not installation
```
Comment (Gautier) [inline basics/searching-packages.md]: "On the command, line the `nix-env` command supports not only installation but also search"

File: basics/searching-packages.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+but also search:
+
+```bash
+nix-env -qaP | grep hello
+```
+
+The flag `-qaP` stand for `--query --available --attr-path`, telling
```
Comment (Gautier) [inline basics/searching-packages.md]: "stands for"

File: basics/searching-packages.md (hunk 1 above)
Comment (Keats) [inline basics/searching-packages.md]: i would rephrase it totally as pretty much every package manager command i know both supports installation and search

File: basics/searching-packages.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+`nix-env` to query all available packages, and to print their
+attribute path.
```
Comment (Keats) [inline basics/searching-packages.md:14]: what's an attribute path?

File: basics/searching-packages.md (hunk 1 above)
Comment (teh) [inline basics/searching-packages.md]: rephrased

File: basics/searching-packages.md (hunk 2 above)
Comment (teh) [inline basics/searching-packages.md]: changed to "the flags" that is what I meant

File: basics/searching-packages.md (hunk 3 above)
Comment (teh) [inline basics/searching-packages.md:14]: added first glossary entry

**Comment by teh** [comment]
PTAL

**Comment by Gautier** [comment]
Looks good to me, the glossary item works well when the book is built
